functions/ssrapp.js

```diff
@@ -44,6 +44,8 @@ app.use(helmet({
 }));
 app.use(cookieParser());
 app.use(setNoCacheHeader);
-app.use(nuxt.render);
-
+app.use(async (req, res) => {
+  await nuxt.ready();
+  nuxt.render(req, res);
```

Comment:
https://nuxtjs.org/api/nuxt-render/
seems `nuxt.render` returns a promise,
should await too?

Reply:
awaiting it would not have actual meaning since express middleware is controlled by `next()` instead of promise. `next()` should be called by `nuxt.render()`

Reply:
Personally I prefer `return nuxt.render(req.res);` at the end to keep any possible semantic